reconcile/gitlab_owners.py

```diff
@@ -43,15 +43,20 @@ def get_change_owners_map(self):
         change_owners_map = dict()
         paths = self.gitlab.get_merge_request_changed_paths(self.mr.iid)
         for path in paths:
-            path_owners = self.owners.get_path_owners(path)
-            if not path_owners:
+            owners = self.owners.get_path_owners(path)
+            path_approvers = owners['approvers']
+            path_reviewers = owners['reviewers']
```

Comment:
will both always exist?

Reply:
At this point, yes. The `utils.repo_owners.RepoOwners()._get_owners_map()` below will give them an empty list case the section is missing in the file.